app/todo_lists/resources.py

```diff
@@ -0,0 +1,71 @@
+import json
+
+from flask import request
+from flask_restful import Resource
+
+from app import db, TodoList
+from app.lib.response_util import buildOkResponse
+
+
+def _returnTodoList(todoList):
+    """Private method to convert a todoList object into a dictionary.
+    """
+    return {
+        "id": todoList.id,
+        "name": todoList.name
+    }
+
+
+class TodoListMultiResource(Resource):
+    """Class for creating and accessing lists.
+    """
+    def get(self):
+        """Method gets a list of todoLists and returns them in an Ok response.
+        """
+        todoLists = TodoList.query.all()
+
+        return buildOkResponse([_returnTodoList(todoList) for todoList in
+                               todoLists])
```

Comment:
Small nit. `todoLists` should start right underneath the above `_` (unless your linter is telling you otherwise).
```
return buildOkResponse([_returnTodoList(todoList) for todoList in
                        todoLists])
```